History Command Negative Repeat count

**Describe the bug**
Program panics when you use our history command and the command is too long
**To Reproduce**
1. Open gosh
2. type very long command
3. run `history`
4. see that it crashes
**Expected behavior**
Shows command (or at least does not crash)
**Additional Information**
<!-- For instance, relevant logs from terminals or the browser's devtool console. -->
```
panic: strings: negative Repeat count

goroutine 1 [running]:
strings.Repeat(0x5dc460, 0x1, 0xfffffffffffffff9, 0xfff, 0xc0003b1c40)
        /home/gitpod/go/src/strings/strings.go:533 +0x5ca
gosh/pkg.History()
        /workspace/gosh/pkg/history.go:32 +0x26c
gosh/pkg.Shell()
        /workspace/gosh/pkg/shell.go:91 +0x959
main.main()
        /workspace/gosh/main.go:11 +0xff
```

--- a/internal/history.go
+++ b/internal/history.go
@@ -29,7 +29,10 @@ func History() {
 	fmt.Println(" ╭━━━━━━━━━━━━━━━━━━━╮")
 	for scanner.Scan() {
 		z := 14 - len(scanner.Text())
-		spaces := strings.Repeat(" ", z)
+		spaces := ""
+		if z >= 0 {
+		  spaces = strings.Repeat(" ", z)
+		}
 		if strings.Compare(string(scanner.Text()), "") == 0 {
 			continue
 		}

